smoketest/scripts/cli/test_container.py

```diff
@@ -80,6 +80,7 @@ def test_basic(self):
 
         self.cli_set(base_path + ['name', cont_name, 'image', cont_image])
         self.cli_set(base_path + ['name', cont_name, 'allow-host-networks'])
+        self.cli_set(base_path + ['name', cont_name, 'sysctl', 'parameter', 'kernel.msgmax', 'value', '8192'])
```

Comment:
Please not only set this value, but also validate it's properly set inside the container.

Reply:
added a check